### ISSUE TITLE
bug when active record is disabled

### ISSUE THREAD

**Issue by dcu** [wvanbergen/request-log-analyzer#62]
rails generates the following format when active record is disabled:

```
Completed in 597ms (View: 298 | 200 OK [http://shapado.com]
```

which is not parsed correctly by rla, I think a regexp like the following should work correctly:

```
/Completed in (\d+)ms \((?:View: (\d+))?(, DB: (\d+)\))? \| (\d\d\d).+\[(http.+)\]/
```

**Comment by wvanbergen** [comment]
Fixed, and released as version 1.9.2

**Comment by dcu** [comment]
thank you very much
